src/web/js/output-ui.js

```diff
@@ -592,17 +376,15 @@
     var goldenAngle = 2.39996322972865332;
     var lastHue = 0;
     
-    function makeSrclocAvaliable(runtime, editors, srcloc) {
+    function makeSrclocAvaliable(runtime, documents, srcloc) {
```

Comment:
This must be called in Pyret stack contexts, right?  Due to the use of `pauseStack` below.  Making a note when I see functions like this.